docs/team/zhangyi.md

```diff
@@ -0,0 +1,40 @@
+---
+layout: page
+title: Zhang Yi's Project Portfolio Page
+---
+
+## Project: tCheck
+
+tCheck is a desktop application used by bubble tea store managers for keeping inventories and keep information of his/her employees. The user interacts with it using a CLI, and it has a GUI created with JavaFX. It is written in Java, and has about 10 kLoC.
+
+Given below are my contributions to the project.
```

Comment:
I think this page will be continually updated, so we should include what we have done to the project so far - which would be the documentation section.

Reply:
I agree :)

Reply:
Noted, will edit accordingly.